src/services/filterExpensesByQuery.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+const filterByUserId = (expenses, userId) => {
+  if (userId) {
+    return expenses.filter(item => item.userId === +userId);
```

Comment:
```suggestion
    return expenses.filter(item => item.userId === Number(userId));
```
The same in other places